### ISSUE TITLE
Perlcritic Runtime Status Not yet activated.

### ISSUE THREAD

**Issue by dirk-seynhaeve** [sfodje/perlcritic#28]
I just noticed that I was writing perfect Perl... Until I ran Perlcritic from the command line on my source files...
It turns out that the Perlcritic VS Code extension no longer reports problems (tested on WSL Remote, SSH Remote (Windows to Linux), and Linux)...

In all cases, I'm getting absolute silence when editing a Perl file (with the `.pl` suffix).

When I go look at the extension, it indicates that it is Installed, Enabled, but not Activated?!

I thought that Perlcritic automatically activates when editing the Perl Language???

**Comment by dirk-seynhaeve** [comment]
There hasn't been any activity on this repository.
@sfodje / @sfodje-ship : is this extension still supported?

**Comment by sfodje-ship** [comment]
No, I'm not actively maintaining the repo, but I'll accept any PRs.

**Comment by leonid-deriv** [comment]
I understand that repo is not actively maintained. But can you answer to the question above? About not activated status.
@dseynhae were you able to "fix" it? 
I am sure I managed to run the extension but recently it stopped working.

**Comment by dirk-seynhaeve** [comment]
@leonid-deriv : I started using a different extension; But I haven't touched Perl for a while now, so I have no idea what configuration would work or not work these days.
